Jobs.Fetcher.TikTok/Jobs/PostsQuery.cs

```diff
@@ -39,6 +39,9 @@ public class PostsQuery : AbstractTikTokFetcher {
                             newAuthor = ApiDataFetcher.GetTikTokAuthorFromAuthorJson(post["author"]);
                         }
                         DbWriter.WriteAuthor(newAuthor, dbContext, logger);
+                    }catch (NullReferenceException) {
+                        Logger.Error("Could not get author data. Skipping this post.");
+                        continue;
                     }catch (Exception e) {
                         Logger.Error("Could not get author data.");
                         throw e;
```

Comment:
I was just seeing this the other day : https://stackoverflow.com/questions/881473/why-catch-and-rethrow-an-exception-in-c

Basically, `throw e` is right in Java (and the most intuitive thing for me), but evil in C#, because it drops the stacktrace. To rethrow an error, you should simply use `throw;`